src/lib.rs

```diff
@@ -260,7 +266,7 @@ fn bmp_size<R: BufRead + Seek>(reader: &mut R) -> ImageResult<ImageSize> {
 
 fn gif_size(header: &[u8]) -> ImageResult<ImageSize> {
     Ok(ImageSize {
-        width:  ((header[6] as usize) | ((header[7] as usize) << 8)),
+        width: ((header[6] as usize) | ((header[7] as usize) << 8)),
```

Comment:
To minimize the diff I'd appreciate removing smaller format only changes like this. I can run a cargo fmt at some point in the future.

Not going to tag every single instance, but there are lots of diffs with just spacing changes. For reference I count 11 blocks of them including this one.